packages/plop/README.md

```diff
@@ -0,0 +1,3 @@
+This is the main source code for the `plop` package. 
+
+For documentation, please refer to [our website](https://plopjs.com/) or [our main README at the root of the repository](../../README.md).
```

Comment:
Up for suggestions about how to communicate this better. This should only ever be seen by contributors on their machine or on GitHub.com. 

Otherwise, our `prepublishOnly.js` script should replace this with the `README.md` that matches our existing `README.md`. 

The reason for two README.md files is so that we can still display a proper documented README for newcomers to the repo, while still providing a README for contributors pointing them to the right place

Reply:
To avoid the need for the switch back and forth, could a git hook that syncs these help?

Or commit to having two copies of some of the information and work on making the main readme talk about all of the projects? I kinda like having a separation between the monorepo readme and the published ones. A markdown comment reminding folks to make their changes in two locations.

Reply:
I'm actually not a huge fan of having two different READMEs. The contributors will forget, we'll forget, then we have to manually squash things, it's kinda a mess IMO.

While I usually like having a separate monorepo README and published one, I think it makes sense here. Open for alternative READMEs after this, but I think this is probably the right move for now